Core/NoOpRunner.Core/Configurators/GameStateConfigurator.cs

```diff
@@ -124,7 +124,10 @@ public IGameStateConfigurator AddPowerUp(PowerUps type, Func<IReadOnlyList<BaseS
 
             var shape = action(Platforms.AsReadOnly());
 
+            //For now
```

Comment:
No, don't leave it like this. Fix `AddPowerUp` instead.
It should probably take just `Func<IReadOnlyList<BaseShape>, PowerUp> action` as an arg, which takes the list of platforms and returns a created powerup. Then just add this returned powerup to the appropriate container.

In NoOpRunner.cs, call the func once for each powerup, passing i.e.
```
platforms =>
                {
                    var platform = platforms.First(p => p.GetType() == typeof(ImpassablePlatform));
                    return new PowerUp(platform.CenterPosX+2, platform.CenterPosY + 1, PowerUps.Invulnerability);
                }
```